### PR TITLE
Fix undefined error when wrong arguments are passed

### PR DESCRIPTION
**Pull request by wakibi**
Should be now able to gracefully return an error instead of breaking the whole application

### CODE REVIEW

File: src/db.ts [hunk 1]
```diff
@@ -23,12 +23,12 @@ interface FetchOptions {
 
 const initOptions = {
   schema(dc: any) {
-    if ((dc as string).includes('kenya')) {
+    if (dc === null || dc === undefined) {
```
Comment (edwinmp) [inline src/db.ts]: Write it as:
`if (!dc) {...}`

File: src/db.ts [hunk 2]
```diff
@@ -63,21 +63,24 @@ export class DB {
   db: IDatabase<any>;
 
   constructor(schemas: string | null) {
-    if (schemas === null) {
-        this.db = this.pgPromise(this.configs, schemas);
+    if (schemas === null || schemas === undefined) {
```
Comment (edwinmp) [inline src/db.ts]: Same here ...

**Review by edwinmp — CHANGES_REQUESTED** [review]
Finish it off with adding a `try & catch` block to surround the code in each route ... this is in the `src/routes/index.ts`. That should handle any & all unexpected errors gracefully.

File: src/db.ts (hunk 2 above)
Comment (wakibi) [reply src/db.ts]: c360959

**Review by edwinmp — APPROVED** [review]

**Pull request merged**